client/src/Components/CardContainer/index.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable operator-linebreak */
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Empty, Pagination, Typography } from 'antd';
+
+import Card from '../Card';
+
+import './style.css';
+
+const { Title } = Typography;
+const cardContainer = ({ title, data, ...rest }) => {
+  const numEachPage = 8;
+  const [limit, setLimit] = useState([0, numEachPage]);
+
+  return (
+    <div className="container" {...rest}>
+      <Title id="container-title" level={2}>
+        {title}
+      </Title>
+      <div className="card-container">
+        {data && data.length !== 0 ? (
+          data
+            .slice(limit[0], limit[1])
+            .map((item) => (
+              <Card
+                key={item.id}
+                id={item.id}
+                Name={item.username}
+                priceByHour={item.price_hour}
+                ImageSrc={item.cover_image}
+                avatarImage={item.avatar}
+                TitleJob={item.title}
+                city={item.location}
+                rate={item.rating}
+                descriptions={item.bio}
```

Comment:
why you passing the values as single values, passing all as an object
```
<Card key={item.id}  provider={item} />
```

Reply:
done